hello_nexus/basic/handler/db_client.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+
+class MyDBClient:
```

Comment:
I would be careful encouraging users to make DB calls in sync handlers. Sync handlers should typically be calling into Temporal APIs. The reason being that the caller controls retrying nexus start operation requests and if there's a chance of failure, we recommend deferring the action to be done in an activity (standalone activities will complete the picture here).
I think there is value in showing dependency injection in the sample, just warn users in comments about the retry caveat.